src/Cron/DayOfMonthField.php

```diff
@@ -70,6 +70,21 @@ public function isSatisfiedBy(DateTime $date, $value)
             return $fieldValue == $date->format('t');
         }
 
+        // Find out if this is the last workday of the month
+        if ( $value == 'D' ) {
```

Comment:
@staabm Added this notice in comments now. Did a force-push of this branch as not to make any unnecessary commits.